src/Bicep.Core/Parsing/Parser.cs

```diff
@@ -71,6 +71,7 @@ public override ProgramSyntax Program()
                         {
```

Comment:
Please update the pseudo-grammar file https://github.com/Azure/bicep/blob/main/docs/grammar.md with the changes

Reply:
I had left the grammar unchanged intentionally. Do we want to document this feature while it's still behind a flag?

Reply:
I think it makes sense to - the grammar doc is a developer guide, not a Bicep user guide.